src/plugins/shared_dictionaries_plugin/shared_dictionaries_plugin_settings.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "hyrise.hpp"
+
+namespace opossum {
+
+/*
+ * The following settings are currently hard coded for the plugin but could be made configurable:
+ *   - option to disable auto start of compression
+ *     The plugin currently starts the merging of dictionaries automatically once it is loaded.
+ *   - option to disable tracking of compression statistics
+ *     The plugin currently tracks memory usage statistics for merged data.
+ *   - option to select all tables / include tables / exclude tables
+ *     The plugin currently tries to merge dictionary segments of all available tables.
+ */
+
+/**
+ * Threshold for the similarity metric between dictionaries for merging
+ */
+class JaccardIndexThresholdSetting : public AbstractSetting {
+ public:
+  JaccardIndexThresholdSetting() : AbstractSetting("Plugin::SharedDictionaries.jaccard_index_threshold") {}
+  const std::string& description() const;
+  const std::string& display_name() const;
+  const std::string& get();
+  void set(const std::string& value);
+
+  std::string _value = "0.1";
```

Comment:
In the PR description, the default value of the jaccard index is set to 0.4. Is the default value of 0.1 intentional?

Reply:
Fixed in https://github.com/hyrise/hyrise/pull/2394/commits/7f88317d0867f9bfd92688f5061daea73a39be35